src/content/apis/currency/financial-modeling-prep.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Financial Modeling Prep API
+description: The most accurate financial data API out there.
+categories:
+  - currency
+features:
+  https: true
+  pricing: paid
```

Comment:
It seems they have free tier as well 🙏 
<img width="999" alt="Screen Shot 2022-10-30 at 5 32 19 PM" src="https://user-images.githubusercontent.com/23402178/198876769-f3676360-8d46-4c03-b20c-06adb08b7600.png">

Reply:
It is for personal use only...

Reply:
People might be using it for personal projects as well right? So I believe it will be nice if we mark it as `upgradable` instead of `paid` 🙏

Reply:
Done